### ISSUE TITLE
⌨ `SignInPage` insufficient color contrast

### ISSUE THREAD

**Issue by mnigh** [GCTC-NTGC/gc-digital-talent#17678]
### ⌨ Accessibility and Expected Behaviour

"Using a personal phone number will help ensure you don't lose access if you change jobs" text has insufficient color contrast in light mode.

### WCAG 2.1 Criteria

https://dequeuniversity.com/rules/axe/4.12/color-contrast?application=axeAPI

### 📋 Steps to Reproduce

1. `pnpm storybook`
2. https://61e099a1bb1465003a73d3ce-bfbhhthzbb.chromatic.com/?path=/story/pages-auth-signinpage--default&globals=theme:GCDT+Light
3. Accessibility tab has violation 

### 🕵️ Details

https://github.com/GCTC-NTGC/gc-digital-talent/blob/a1e3f2797d50b8c3a31fcbc20f7e2af7bff46469/apps/web/src/pages/Auth/SignInPage/SignInPage.tsx#L388

### 📸 Screenshot

<img width="1509" height="866" alt="Image" src="https://github.com/user-attachments/assets/9207449c-f33b-4a43-8f90-71bc738fdd5f" />

### 🙋‍♀️ Proposed Solution

Change from `text-gray-500` to `text-gray-600` to be in line with other similar text.

### ✅ Acceptance Criteria

- [ ] `SignInPage.Default` story does not have a11y violations
